components/Footer/Footer.tsx

```diff
@@ -131,16 +156,23 @@ const AccountLinks = ({ authenticated, user, signOut }: PageFooterProps) => {
         Sign Out
       </NavLink>
     </>
-  ) : null
+  ) : (
+    <NavLink
+      handleClick={() => console.log("SignIn clicked")}
```

Comment:
Once #893 Is in, you'll be able to trigger the [sign in modal with `dispatch(authStepChanged("start"))`](https://github.com/codeforboston/maple/pull/893/files#diff-d8e81a1fcdd5e9228df2e589ce5afa959f72897eb4378cf89593a32f710edb77R16)

Reply:
will wait for 893.

Reply:
The build is failing after i merged my code with the code for boston main. I have looked at the code and it looks ok. Do you mind taking a second look?